### PR TITLE
add leo-ntp icons

### PR DESCRIPTION
**Pull request by CumpsD**
## Proposed change

Add icons for LeoNTP, see [leobodnar.com](https://www.leobodnar.com/shop/index.php?main_page=product_info&cPath=120&products_id=365) for logo and icon. (Custom integration is in development)

## Type of change

- [ ] Add a new logo or icon for a new core integration
- [ ] Add a missing icon or logo for an existing core integration
- [x] Add a new logo or icon for a custom integration (custom component)
  - [ ] I've opened up a PR for my custom integration on the [Home Assistant
    Python wheels repository](https://github.com/home-assistant/wheels-custom-integrations)
- [ ] Replace an existing icon or logo with a higher quality version
- [ ] Removing an icon or logo

## Additional information

- This PR fixes or closes issue: fixes #
- Link to code base pull request: https://github.com/CumpsD/home-assistant-leo-ntp
- Link to documentation pull request: 
- Link to integration documentation on our website: 

## Checklist

- [x] The added/replaced image(s) are **PNG**
- [x] Icon image size is 256x256px (`icon.png`)
- [x] hDPI icon image size is 512x512px for  (`icon@2x.png`)
- [x] Logo image size has min 128px, but max 256px, on the shortest side (`logo.png`)
- [x] hDPI logo image size has min 256px, but max 512px, on the shortest side (`logo@2x.png`)

### CODE REVIEW

**Review by frenck — APPROVED** [review]
Hi there @CumpsD 👋 

I could not find the public repository providing this custom integration, could you point me to it?

Thanks! 👍 

../Frenck

**Review by frenck — CHANGES_REQUESTED** [review]
Oop, wrong button, let me try again :)

../Frenck

**Comment by CumpsD** [comment]
> Hi there @CumpsD 👋
> 
> I could not find the public repository providing this custom integration, could you point me to it?
> 
> Thanks! 👍
> 
> ../Frenck

It's in the PR: https://github.com/CumpsD/home-assistant-leo-ntp

But it is not released yet, since apparently HACS wants the images in this brands repo first :)

**Review by frenck — APPROVED** [review]
> But it is not released yet, since apparently HACS wants the images in this brands repo first :)

That is fine, we don't require HACS (integrations can be distributed without HACS as well), I just needed a pointer to the repository.

Looks good! Thanks for contributing 👍 

../Frenck

**Comment by frenck** [comment]
This is now the second time you merge the `master` branch, and I'm not fully understanding why you did that? Was there a merge conflict? 

It fully re-triggers the CI every time, and as I don't see a reason for the action, it feels unneeded.

../Frenck

**Comment by CumpsD** [comment]
> This is now the second time you merge the `master` branch, and I'm not fully understanding why you did that? Was there a merge conflict?
> 
> It fully re-triggers the CI every time, and as I don't see a reason for the action, it feels unneeded.
> 
> ../Frenck

I thought it had to be up to date with `master` to get merged, since you approved it and all checks were ok but nothing happened

**Comment by frenck** [comment]
> I thought it had to be up to date with master to get merged

There is no such requirement?

>  since you approved it and all checks were ok but nothing happened

I can't merge it when builds are still running. At the time I approved it, CI was in progress. Right now I came back to merge, and CI is again still in progress (due to your merge).

Right now, I'll go offline, so will not be able to look again (and thus this PR will stay pending).

Besides that, it just wastes CI resources/servers and CPU cycles, which is not good for the environment 🌱

../Frenck

**Comment by CumpsD** [comment]
I didn't know, won't touch it again

**Comment by CumpsD** [comment]
> > I didn't know, won't touch it again
> 
> Don't worry about it ❤️

Thanks!
